tests/Fixtures/App/config/config.php

```diff
@@ -12,4 +12,4 @@
 $container->setParameter('cmf_testing.bundle_fqn', 'Symfony\Cmf\Bundle\ContentBundle');
 $loader->import(CMF_TEST_CONFIG_DIR.'/default.php');
 $loader->import(CMF_TEST_CONFIG_DIR.'/phpcr_odm.php');
-$loader->import(__DIR__.'/cmf_content.yml');
+$loader->import(__DIR__ . '/cmf_content.yml');
```

Comment:
the codestyle asks to not have whitespace around the `.` in string concatentation